WooCommerce/Resources/Images.xcassets/blaze.imageset/Contents.json

```diff
@@ -1,7 +1,7 @@
 {
   "images" : [
     {
-      "filename" : "blaze.pdf",
+      "filename" : "temp-flame-b6.pdf",
```

Comment:
since the existing Blaze entry point in the Menu tab also uses this asset (hard to see the image difference in the PR), I also tested the Menu tab. It looks like the icon got smaller in the Menu tab, not sure if it's intentional?

before | after
-- | --
![Simulator Screenshot - iPhone 14 - 2023-06-29 at 09 58 06](https://github.com/woocommerce/woocommerce-ios/assets/1945542/f73e0e6c-f41b-4844-a01a-74cc9a94aaf7) | ![Simulator Screenshot - iPhone 14 - 2023-06-29 at 09 59 19](https://github.com/woocommerce/woocommerce-ios/assets/1945542/6183290a-d989-4d6e-97c9-b1873045cb95)

Reply:
Good catch! It seems the icon has some padding around so it's smaller in the menu. I asked for help exporting the icon without the padding [mBhyoV7OPhFnOugxLy6SMS-fi-82:10255], will update this in a future PR.